torch/functional.py

```diff
@@ -1234,23 +1235,21 @@ def norm(input, p="fro", dim=None, keepdim=False, out=None, dtype=None):  # noqa
             vector norm will be calculated, if it is 2-tuple of ints, matrix norm
             will be calculated. If the value is None, matrix norm will be calculated
```

Comment:
But what happens if it has 3 or more dimensions? Also, it might be helpful to specify what the input to the norm is. For example, if dim is an integer then the vector norm along that dimension is computed. If dim is a 2-tuple then the matrix norm along those dimensions is computed. If dim is None and the tensor is 1D then ...